### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by rozmansi**
Hi Jeroen,

I couldn't help myself to get involved in some Go coding after a looong time. 😉

Regards, Simon

### CODE REVIEW

File: internal/failover/ping_default.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//go:build !windows
+
+package failover
+
+import (
+	"net"
```
Comment (jwijenbergh) [inline internal/failover/ping_default.go:6]: missing imports
```
"github.com/go-errors/errors"
"golang.org/x/net/icmp"
```

File: internal/failover/ping_windows.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package failover
```
Comment (jwijenbergh) [inline internal/failover/ping_windows.go:1]: should we add:

```go
//go:build windows
```

File: internal/discovery/.gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/organization_list.json
+/organization_list.json.tmp.minisig
```
Comment (jwijenbergh) [inline internal/discovery/.gitignore]: Any reason why these contain `.tmp`?

File: internal/failover/ping_windows.go (hunk 2 above)
Comment (rozmansi) [reply internal/failover/ping_windows.go:1]: No need to. It's Go's convention if the filename ends with `_windows.go` to compile it when `GOOS=windows` only.

File: internal/discovery/.gitignore (hunk 3 above)
Comment (rozmansi) [reply internal/discovery/.gitignore]: Ouch. A leftover from my build system... The Makefile produces temporary files, and only after Minisign verifies them, moves the .json file to its final name. This prevents `make` from misinterpreting the .json file in this folder as being verified on its next invocation. Will cleanup...

File: internal/failover/ping_default.go (hunk 1 above)
Comment (rozmansi) [reply internal/failover/ping_default.go:6]: Missed this one as `go build` didn't touch this file on my computer. 😇 Fixing...

File: internal/discovery/.gitignore (hunk 3 above)
Comment (rozmansi) [reply internal/discovery/.gitignore]: Actually, I removed this completely from this PR, as this `.gitignore` is entirely specific for my build system.

**Comment by jwijenbergh** [comment]
Thanks!

**Pull request merged**